src/VirtualClient/VirtualClient.Actions/MLPerfTraining/MLPerfTrainingExecutor.cs

```diff
@@ -0,0 +1,517 @@
+        /// <summary>
+        /// Executes the MLPerf Training workload.
+        /// </summary>
+        protected override async Task ExecuteAsync(EventContext telemetryContext, CancellationToken cancellationToken)
+        {
+            this.PrepareBenchmarkConfigsAndScenarios();
+            string shardsPath = this.PlatformSpecifics.Combine("/mlperftraining0", $"{this.DataPath}", "mlperf-training-package", "hdf5", "training-4320");
+            string evalPath = this.PlatformSpecifics.Combine("/mlperftraining0", $"{this.DataPath}", "mlperf-training-package", "hdf5", "eval_varlength");
+            string checkpointPath = this.PlatformSpecifics.Combine("/mlperftraining0", $"{this.DataPath}", "mlperf-training-package", "phase1");
+
+            using (BackgroundOperations profiling = BackgroundOperations.BeginProfiling(this, cancellationToken))
+            {
+                string execCommand = $"su -c \"source {this.ConfigFile}; " + 
+                                     $"env BATCHSIZE={this.BatchSize} " + 
+                                     $"DGXNGPU={this.GPUNum} " + 
+                                     $"CUDA_VISIBLE_DEVICES=\"{this.GetGPULabels()}\" " + 
+                                     $"CONT={this.GetContainerName()} DATADIR={shardsPath} DATADIR_PHASE2={shardsPath} EVALDIR={evalPath} CHECKPOINTDIR={checkpointPath} CHECKPOINTDIR_PHASE1={checkpointPath} ./run_with_docker.sh\"";
+
+                using (IProcessProxy process = await this.ExecuteCommandAsync("sudo", execCommand, this.ExecutionDirectoryPath, telemetryContext, cancellationToken)
+                   .ConfigureAwait())
+                {
+                    if (!cancellationToken.IsCancellationRequested)
+                    {
+                        await this.LogProcessDetailsAsync(process, telemetryContext)
+                            .ConfigureAwait();
+
+                        process.ThrowIfErrored<WorkloadException>(errorReason: ErrorReason.WorkloadFailed);
+
+                        await this.CaptureMetricsAsync(process, telemetryContext, cancellationToken, MLPerfTrainingExecutor.AccuracySummary)
+                            .ConfigureAwait();
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Parse metrics and push to telemetry
+        /// </summary>
+        /// <param name="process">Execute process for StandardOutput containing the metrics</param>
+        /// <param name="telemetryContext"></param>
+        /// <param name="cancellationToken"></param>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        private async Task CaptureMetricsAsync(IProcessProxy process, EventContext telemetryContext, CancellationToken cancellationToken, string context = null)
+        {
+            // Convert StandardOutput to string
+            ConcurrentBuffer buffer = process.StandardOutput;
+
+            string logs = string.Concat(buffer.ToString(), Environment.NewLine);
```

Comment:
just wondering, why is this line required?

Reply:
It is redundant. I have removed the line